packages/mui-base/src/TabPanelUnstyled/TabPanelUnstyled.types.ts

```diff
@@ -14,7 +14,7 @@ export interface TabPanelUnstyledOwnProps {
   /**
    * The value of the TabPanel. It will be shown when the Tab with the corresponding value is selected.
    */
-  value: number | string;
+  value?: number | string;
```

Comment:
This PR actually relaxes the requirement to have `value` defined in TabPanels (still required for SSR, though).